shell-switcher.el

```diff
@@ -277,8 +277,8 @@ If OTHER-WINDOW is nil (the default), change the current window.
 If OTHER-WINDOW is t, change another window."
   (if (-shell-exist-p)
       (if other-window
-	  (switch-to-buffer-other-window (-most-recent) t)
-	(switch-to-buffer (-most-recent) t))
+          (switch-to-buffer-other-window (-most-recent) t)
```

Comment:
My indentation is consistent with the indentation used on `(if` forms in the rest of your file, 4 spaces.  Feel free to put it back to 3 spaces if you wish.